website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -136,40 +136,27 @@ After modifying the `*_types.go` file always run the following command to update
 $ make generate
 ```
 
-The above makefile target will invoke the [controller-gen][controller_tools] utility to update the `api/v1alpha1/zz_generated.deepcopy.go` file to ensure our API's Go type definitons implement the `runtime.Object` interface that all Kind types must implement.
+**Note**: The above makefile target will invoke the [controller-gen][controller_tools] utility to update the `api/v1alpha1/zz_generated.deepcopy.go` file to ensure our API's Go type definitons implement the `runtime.Object` interface that all Kind types must implement.
 
-### Generating CRD manifests
+## Generating CRD manifests
 
 Once the API is defined with spec/status fields and CRD validation markers, the CRD manifests can be generated and updated with the following command:
 
 ```console
 $ make manifests
 ```
 
-This makefile target will invoke controller-gen to generate the CRD manifests at `config/crd/bases/cache.example.com_memcacheds.yaml`.
-
-#### OpenAPI validation
```

Comment:
moved to reference sine we do not use OpenApi validation in this example.

Reply:
Is this section being referenced elsewhere then? OpenAPI validation is really important to discuss.